src/nanosvg.h

```diff
@@ -1322,12 +1322,32 @@ NSVGNamedColor nsvg__colors[] = {
 static unsigned int nsvg__parseColorName(const char* str)
 {
 	int i, ncolors = sizeof(nsvg__colors) / sizeof(NSVGNamedColor);
-
-	for (i = 0; i < ncolors; i++) {
+	
+	for (i = 0; i < 10; i++) {
 		if (strcmp(nsvg__colors[i].name, str) == 0) {
 			return nsvg__colors[i].color;
 		}
 	}
+	if (ncolors > 10)
+	{
```

Comment:
Addressed.(I don't quite have a handle on this UI yet - the update automatically goes into the same pull request, I don't create a new one, is that right?)  

    On Friday, 2 September 2016, 18:22, Mikko Mononen <notifications@github.com> wrote:
 

 In src/nanosvg.h:>  		if (strcmp(nsvg__colors[i].name, str) == 0) {
>  			return nsvg__colors[i].color;
>  		}
>  	}
> +	if (ncolors > 10)
> +	{
curlies on the same line with if/while/for—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.